packages/config/config/devices/0x024f/fwc01.json

```diff
@@ -0,0 +1,227 @@
+{
+	"manufacturer": "Smartly AS",
+	"manufacturerId": "0x024f",
+	"label": "FWC01",
+	"description": "Wheel Controller",
+	"devices": [
+		{
+			"productType": "0x0003",
+			"productId": "0x1003"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": [
+		{
+			"#": "4",
+			"label": "Safety Activate Delay",
+			"unit": "seconds",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 32767,
+			"defaultValue": 0
+		},
+		{
+			"#": "5",
+			"label": "Safety OFF Period",
+			"unit": "seconds",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 32767,
+			"defaultValue": 600
+		},
+		{
+			"#": "6",
+			"label": "Safety ON Period",
+			"unit": "seconds",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 32767,
+			"defaultValue": 300
+		},
+		{
+			"#": "10",
+			"label": "Dimming speed",
+			"description": "Dimmer Speed in 1/100th of a Second",
+			"unit": "/100s",
+			"valueSize": 2,
+			"minValue": 30,
+			"maxValue": 32767,
+			"defaultValue": 300
+		},
+		{
+			"#": "11",
+			"label": "Dimmer mode",
+			"description": "Dimmer Operation Mode",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1
+		},
+		{
+			"#": "12",
+			"label": "Maximum Level",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 99,
+			"defaultValue": 90
+		},
+		{
+			"#": "13",
+			"label": "Minimum Level",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 99,
+			"defaultValue": 15
+		},
+		{
+			"#": "14",
+			"label": "UI modes and Zones",
+			"description": "Enable UI Modes and Zones via Bits",
```

Comment:
This sounds like the parameter should be converted to [partials](https://zwave-js.github.io/node-zwave-js/#/config-files/partial-parameters) (one per bit)

Reply:
Yes, that looks like a very good way to define these options. I will fix it.